source/code/plugins/tailfilereader.rb

```diff
@@ -22,17 +22,17 @@ def initialize(paths)
       @log.formatter = proc do |severity, time, progname, msg|
         "#{severity} #{msg}\n"
       end
-      @log.info "Received paths from sudo tail plugin : #{paths}, log_level=#{level}"
+      @log.info "sudo tail plugin in agent configuration is provisioned to tail files from the following paths: #{paths}, log_level=#{level}"
     end
 
     attr_reader :paths
 
     def file_exists(path)
       if File.exist?(path)
-        @log.info "Following tail of #{path}"
+        @log.info "File path #{path} exists. Trying to tail."
         return path
       else
-        @log.warn "#{path} does not exist. Cannot tail the file."
+        @log.info "#{path} does not exist or not accessable. Cannot tail the file. Skipping."
```

Comment:
Why is this (and other similar lines) changed from log.warn => log.info?

Reply:
we want this message to be logged in default logs. Currently to enable debug logging , cx needs to make changes in their config files manually.

Reply:
Info lines are included by default, but not warnings?

Reply:
yes